src/components/Block/ContactPicker.vue

```diff
@@ -28,16 +14,19 @@ const emit = defineEmits<{
 }>();
 
 const { account } = useAccount();
+const contactsStore = useContactsStore();
 
 const allContacts = computed(() => {
-  if (!account) return CONTACTS;
+  if (!account) return contactsStore.contacts;
+  if (contactsStore.contacts.find(contact => contact.address === account))
+    return contactsStore.contacts;
```

Comment:
nit. when `if` splits into two lines I'd say it makes sense to use `{}` so mistakes are less likely to happen if we ever need to add more statements in if body.